Notebooks/Metrics.ipynb

```diff
@@ -43,6 +43,7 @@
     "import pandas as pd\n",
```

Comment:
<p>What time unit does "mean_travel_time" use now? Hours? Why multiply with 3600? That would only make sense if we had hours and wanted seconds</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AndOleAnd/Capstone_N_A_P/pull/87/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>